packages/fluttium_cli/pubspec.yaml

```diff
@@ -13,8 +13,10 @@ dependencies:
   args: ^2.4.0
   cli_completion: ^0.3.0
   collection: ^1.17.1
-  fluttium_driver: ^0.1.3
-  fluttium_interfaces: ^0.1.0
+  fluttium_driver:
+    path: ../fluttium_driver
+  fluttium_interfaces:
+    path: ../fluttium_interfaces
```

Comment:
Needs to be reverted

Reply:
The [pubspec_overrides.yaml](https://github.com/wolfenrain/fluttium/blob/main/packages/fluttium_cli/pubspec_overrides.yaml) already does this for u 😉

Reply:
The install doesn't work if I revert the changes. Seems like it installs the other packages from pub.dev.

```
flutter pub global activate --source git https://github.com/nohli/fluttium.git --git-path packages/fluttium_cli --git-ref custom-web-port --overwrite
Package fluttium_cli is currently active from Git repository "https://github.com/nohli/fluttium.git".
Downloading packages... . 
  cli_completion 0.3.0 (0.5.1 available)
  file 6.1.4 (7.0.0 available)
~ fluttium_cli 0.1.2 from git https://github.com/nohli/fluttium.git at ee9634 in packages/fluttium_cli (was 0.1.2 from git https://github.com/nohli/fluttium.git at 96f764 in packages/fluttium_cli)
* fluttium_driver 0.1.3 (was 0.1.3 from git https://github.com/nohli/fluttium.git at 96f764 in packages/fluttium_driver)
* fluttium_interfaces 0.1.0 (was 0.1.0 from git https://github.com/nohli/fluttium.git at 96f764 in packages/fluttium_interfaces)
  mason 0.1.0-dev.57 (0.1.0-dev.60 available)
  mason_logger 0.2.16 (0.3.1 available)
  process 4.2.4 (5.0.2 available)
  pub_updater 0.3.1 (0.5.0 available)
Building package executables... (1.2s)
Failed to build fluttium_cli:fluttium:
../../../.pub-cache/git/fluttium-ee963406b8ccad6a4845bfcb85e68a1b60ac4dc0/packages/fluttium_cli/lib/src/commands/test_command/test_command.dart:304:9: Error: No named parameter with the name 'webPort'.
        webPort: _webPort,
        ^^^^^^^
```

Reply:
Yea the git install does not use them but locally it works. I assumed that is how you were testing it, either way we can just remove them from this PR because I always test locally

Reply:
Done ✅